handler/category_dimension_import_test.go

```diff
@@ -126,6 +129,37 @@ func TestHandle(t *testing.T) {
 				So(sleepRandomCalls, ShouldHaveLength, 0)
 			})
 		})
+
+		Convey("When Handle is successfully triggered with only Geography codelists", func(c C) {
+			e := &event.CategoryDimensionImport{
+				InstanceID:     testInstanceID,
+				JobID:          testJobID,
+				DimensionID:    "test-variable",
+				CantabularBlob: testBlob,
+				IsGeography:    true,
+			}
+			msg := kafkaMessage(c, e)
+			err := eventHandler.Handle(ctx, workerID, msg)
+			So(err, ShouldBeNil)
+
+			Convey("Then the corresponding codebook obtained from cantabular should not have Dimension Options", func() {
+				So(ctblrClient.GetDimensionOptionsCalls(), ShouldHaveLength, 0)
+			})
+
+			Convey("Then the corresponding instance is obtained from dataset API", func() {
```

Comment:
This should begin with And instead of Then (as there has already been a Then after the When)
```suggestion
			Convey("And the corresponding instance is obtained from dataset API", func() {
```

Reply:
This has been addressed on this PR: https://github.com/ONSdigital/dp-import-cantabular-dimension-options/pull/46